app/commands/codemod/writer.go

```diff
@@ -0,0 +1,27 @@
+package codemod
+
+import (
+	"io/ioutil"
+
+	"github.com/juju/errors"
+)
+
+func Write(newSRCs []*SRCHunk) error {
+
+	// TODO(waigani) use one open file handler per file to write all changes
```

Comment:
Make sure you add this TODO as an issue, milestone, or in your project management tool; eg. Trello.